Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -154,15 +156,16 @@ public function evaluate()
                     } catch (NoMatchingRouteException) {
                         $this->systemLogger->warning(sprintf('Could not resolve "%s" to a node uri. Arguments: %s', $matches[0], json_encode($uriBuilder->getLastArguments())), LogEnvironment::fromMethodName(__METHOD__));
                     }
-                    $this->runtime->addCacheTag('node', $matches[2]);
+                    $this->runtime->addCacheTag(
+                        CacheTag::forDynamicNodeAggregate($contentRepository->id, $nodeAddress->contentStreamId, NodeAggregateId::fromString($matches[2]))->value
+                    );
```

Comment:
is there a specific reason we use the content stream id here and not the workspace name?

Because im just stumbling over this in the node identity refactoring

Reply:
https://github.com/neos/neos-development-collection/issues/4810

This is the next step